README.md

```diff
@@ -125,7 +132,22 @@ Current state | Next word | Next state
 
 ## A2.4
 
-«replace this with your answer»
+Current State | Next Word | Next State
+--------------|-----------|--------------
+S0            | the       | S1
+S0            | the       | S2
+S1            | lazy      | S1
+S1            | lazy      | S2
+S1            | smelly    | S1
+S1            | smelly    | S2
+S2            | dog       | S3 
+S2            | cat       | S3
+S3            | ate       | S4
+S3            | ran       | S4
+S4            | slowly    | S5
+S4            | noisily   | S5
+S4            | EOI       | END
+S5            | EOI       | END
```

Comment:
The same comments apply to this table. When you found yourself writing two transitions with the same label from one state, you should have stopped and wondered what that could actually mean. FSMs are deterministic, so they don't make guesses.

No extra deduction: you tried to represent your diagram